README.reproducible.md

```diff
@@ -0,0 +1,148 @@
+#Armory Reproducible Build Documentation
+
+##Gitian
+
+Gitian is the build system that Bitcoin Core uses for reproducible builds.
+Armory has adopted it for OS X and Windows reproducible builds. This guide
```

Comment:
Right now the Linux reproducible build just uses the Debian Reproducible Build Toolchain installed in a chroot. It could use Gitian if we want for consistency, but it is already capable of producing reproducible deb packages without Gitian. I think we talked about whether to use Gitian, but I don't recall if we came to a decision. I think Gitian is excessive in this case, but like I said, we may want to use it for consistency.

There is an old Linux Gitian descriptor, so that definitely should be removed or modified to work with the Debian Toolchain.